units/other/evocom/armcomlvl2.lua

```diff
@@ -124,6 +124,7 @@ return {
 			minimum_respawn_stun = 5,
 			distance_stun_multiplier = 1,
 			fall_damage_multiplier = 5,--this ensures commander dies when it hits the ground so effigies can trigger respawn.
+			--stockpilelimit = 3,
```

Comment:
Stockpile limit defined but no stockpile configuration on weapon

Reply:
@SethDGamre Is this supposed to be a stockpiling weapon?

Reply:
Removed for now, if its needed it can be implemented fully another time.